<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountUi</class>
 <widget class="QWidget" name="AccountUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AccountUi</string>
  </property>
  <widget class="QWidget" name="logo" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>fangy14</string>
   </property>
  </widget>
  <widget class="QWidget" name="flowIcon" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="timeIcon" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="moneyIcon" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="flowLabel">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>93</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>流量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>168</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时长</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="moneyLabel">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>余额</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="flowNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading...</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading...</string>
   </property>
  </widget>
  <widget class="QLabel" name="moneyNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>181</width>
     <height>3</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="extendWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="Line" name="line_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>259</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="extendButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
